### PR TITLE
Test changes

### PR DESCRIPTION
**Pull request by Lesenr1**

### CODE REVIEW

File: test/betting/helpers/bootstrap.jsligo [hunk 1]
```diff
@@ -34,13 +34,12 @@ export const bootstrap = () => {
...
   // Boostrapping BETTING contract
-  let betting_path = "../../../src/contracts/jsligo/betting/main.jsligo";
-  let iBis = Test.run( (x : Types.storage) => x, init_storage);
-  let [betting_address, _c, _n] = Test.originate_from_file(betting_path, iBis, 0 as mutez);
-  let betting_taddress = (Test.cast_address(betting_address) as typed_address<parameter_of Betting,Types.storage> );
-  let betting_contract = Test.to_contract (betting_taddress);
+  const betting_path = "../../../src/contracts/jsligo/betting/main.jsligo";
+  const orig : origination_result<parameter_of Betting,Types.storage> = Test.originate_from_file(betting_path, init_storage, 0 as mutez);
+  const betting_contract = Test.to_contract (orig.addr);
+  const betting_address = Test.to_address (orig.addr);
```
Comment (SuzanneSoy) [inline test/betting/helpers/bootstrap.jsligo:40]: :thinking: isn't orig.addr already an address? If not maybe we should fix the API, sounds confusing

File: test/betting/helpers/bootstrap.jsligo [hunk 2]
```diff
@@ -34,13 +34,12 @@ export const bootstrap = () => {
   };
 
   // Boostrapping BETTING contract
-  let betting_path = "../../../src/contracts/jsligo/betting/main.jsligo";
-  let iBis = Test.run( (x : Types.storage) => x, init_storage);
```
Comment (SuzanneSoy) [inline test/betting/helpers/bootstrap.jsligo:38]: This `Test.run` call is missing after your changes

File: test/betting/helpers/callback.jsligo [hunk 3]
```diff
@@ -34,13 +34,10 @@ export const base_storage = (bettingAddr : address) : Callback.storage => {retur
 
 export const originate_from_file = (initial_storage : Callback.storage) : originated => {
     let betting_path           = "../../../src/contracts/jsligo/betting/callback/main.jsligo";
-    let iTres                  = Test.run( (x : Callback.storage) => x, initial_storage);
```
Comment (SuzanneSoy) [inline test/betting/helpers/callback.jsligo:37]: This `Test.run` call is missing after your changes

File: test/oracle/helpers/bootstrap.jsligo [hunk 4]
```diff
@@ -49,10 +49,9 @@ export const bootstrap_oracle = () => {
 
     // Boostrapping Oracle contract
     let oracle_path = "../../../src/contracts/jsligo/oracle/main.jsligo";
-    let iBis = Test.run( (x : Types.storage) => x, init_storage);
```
Comment (SuzanneSoy) [inline test/oracle/helpers/bootstrap.jsligo:52]: Same here

File: test/oracle/helpers/callback.jsligo [hunk 5]
```diff
@@ -25,15 +25,12 @@ export const base_storage : Callback.storage = {
     meta : (Map.empty as map<string, bytes>)
 }
 
-export const originate_from_file = (initial_storage : Callback.storage) => {
-    let oracle_path = "../../../src/contracts/jsligo/oracle/callback/main.jsligo";
-    let iTres = Test.run( (x : Callback.storage) => x, initial_storage);
```
Comment (SuzanneSoy) [inline test/oracle/helpers/callback.jsligo:30]: same here

**Review by SuzanneSoy — APPROVED** [review]
Tentatively approved, asked questions

File: test/betting/helpers/bootstrap.jsligo (hunk 1 above)
Comment (Lesenr1) [reply test/betting/helpers/bootstrap.jsligo:40]: it's a "typed address".. but kinda annoying to have such a long label for it

File: test/betting/helpers/bootstrap.jsligo (hunk 2 above)
Comment (Lesenr1) [reply test/betting/helpers/bootstrap.jsligo:38]: yes, we now perform the run from inside the originate_from_file function

**Review by LigoSuzanneSoy — APPROVED** [review]

**Pull request merged**